### PR TITLE
Create accessing-channel-partner-portal.md

### PR DESCRIPTION
**Pull request by diana-lakatos**

### CODE REVIEW

File: _PlatformOS/getting-started/accessing-channel-partner-portal.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Accessing Our Channel Partner Portal
```
Comment (kv109) [inline _PlatformOS/getting-started/accessing-channel-partner-portal.md]: I believe that the official name is just "Partner Portal". I would replace any occurence of "Channel Partner" to just "Partner" (with capital letter).

File: _PlatformOS/getting-started/accessing-channel-partner-portal.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+permalink: /get-started/accessing-channel-partner-portal
+---
+This guide will help you get access, and log in to our Channel Partner Portal, where you can create, manage and configure Instances (sites), and create and manage the permissions of other Partners (users). 
+
+## Steps 
+
+Accessing our Channel Partner Portal is a three-step process:
+
+*   Get access to our Channel Partner Portal
```
Comment (kv109) [inline _PlatformOS/getting-started/accessing-channel-partner-portal.md]: "Get access to" => "Get invitation to"?

File: _PlatformOS/getting-started/accessing-channel-partner-portal.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+permalink: /get-started/accessing-channel-partner-portal
+---
+This guide will help you get access, and log in to our Channel Partner Portal, where you can create, manage and configure Instances (sites), and create and manage the permissions of other Partners (users). 
+
+## Steps 
+
+Accessing our Channel Partner Portal is a three-step process:
+
+*   Get access to our Channel Partner Portal 
+*   Sign in
```
Comment (kv109) [inline _PlatformOS/getting-started/accessing-channel-partner-portal.md]: "Sign in" => "Set up your password"

File: _PlatformOS/getting-started/accessing-channel-partner-portal.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+* Email: [to be added]() 
+* Call: [+64 1337 128 MPP](tel:+641337128MPP)
+
+You can also get access to our Channel Partner Portal if a registered Channel Partner creates an account for you. 
+
+Once registered, you will get an email invitation. Click on the ```Accept invitation``` link to activate your account. 
+
+Clicking on the link redirects you to a page where you can set your password. 
+
+### Step 2: Sign in
```
Comment (kv109) [inline _PlatformOS/getting-started/accessing-channel-partner-portal.md]: "Sign in" => "Set up your password"

File: _PlatformOS/getting-started/accessing-channel-partner-portal.md (hunk 1 above)
Comment (diana-lakatos) [reply _PlatformOS/getting-started/accessing-channel-partner-portal.md]: Thanks, I'll keep this in mind for all upcoming docs.

File: _PlatformOS/getting-started/accessing-channel-partner-portal.md (hunk 2 above)
Comment (diana-lakatos) [reply _PlatformOS/getting-started/accessing-channel-partner-portal.md]: I went with "Get invited to", thanks!

**Pull request merged**